src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -394,13 +392,21 @@
     <!-- Non-null type is required for this node kind -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
 
+    <!-- Various assignable expressions, including some BoundDeconstructionVariables for nested variables -->
     <Field Name="LeftVariables" Type="ImmutableArray&lt;BoundExpression&gt;"/>
```

Comment:
@AlekseyTs, from discussion with Chuck I will remove this node and store the tree of variables some other way (without creating new kind of bound node).